SU2_CFD/src/solvers/CNEMONSSolver.cpp

```diff
@@ -89,43 +92,55 @@ void CNEMONSSolver::Preprocessing(CGeometry *geometry, CSolver **solver_containe
     SetPrimitive_Gradient_LS(geometry, config);
   }
 
+  if (Output) ompMasterAssignBarrier(nPrimVarGrad, nPrimVarGrad_bak);
+
   /*--- Compute the limiters ---*/
 
   if (muscl && !center && limiter && !van_albada && !Output) {
     SetPrimitive_Limiter(geometry, config);
   }
+  const long unsigned int offset=(nSpecies+2);
+  ComputeVorticityAndStrainMag<1>(*config, iMesh,offset);
```

Comment:
```suggestion
  ComputeVorticityAndStrainMag(*config, iMesh, offset);
```
Matching change to the template comment.

Reply:
Done!